servers/plugins/org.fusesource.ide.server.karaf.core/src/org/fusesource/ide/server/karaf/core/util/KarafUtils.java

```diff
@@ -326,10 +326,12 @@ public static String getBundleSymbolicName(IModule module) throws CoreException
 	
 	public static String stripParametersFromSymbolicName(String symbolicName) {
 		String resVal = symbolicName;
-		// sometimes parameters are added to the symbolic name - we should ignore them
-		int paramIdx = symbolicName.indexOf(';');
-		if (paramIdx != -1) {
-			resVal = symbolicName.substring(0, paramIdx);
+		if (symbolicName != null) {
```

Comment:
might worth to rebase on master to avoid seeing this modification in the Pull Request